### PR TITLE
GDM ansible scripts

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by redhatrises** [comment]
@openscap-ci test this please

File: shared/templates/static/ansible/gnome_gdm_disable_guest_login.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
```
Comment (mpreisler) [inline shared/templates/static/ansible/gnome_gdm_disable_guest_login.yml]: doesn't this require reboot? gdm won't reload this file once it's already running AFAIK

**Comment by shawndwells** [comment]
On 8/11/17 4:53 PM, Martin Preisler wrote:
>
> *@mpreisler* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In shared/templates/static/ansible/gnome_gdm_disable_guest_login.yml
> <https://github.com/OpenSCAP/scap-security-guide/pull/2237#discussion_r132781356>:
>
> > @@ -0,0 +1,14 @@
> +# platform = multi_platform_rhel, multi_platform_fedora
> +# reboot = false
>
> doesn't this require reboot? gdm won't reload this file once it's
> already running AFAIK
>

I don't have a VM with GNOME, but I think gdm-restart will work:
https://help.gnome.org/admin/gdm/stable/binaries.html.en#gdmrestartcommandline

**Comment by shawndwells** [comment]
On 8/11/17 4:53 PM, Martin Preisler wrote:
>
> doesn't this require reboot? gdm won't reload this file once it's
> already running AFAIK
>

update: apparently since RHEL6 Red Hat has not shipped gdm-restart,
instead users have to pkill or telinit:
https://access.redhat.com/solutions/36382

*Technically* we can reload without reboot. But I'm OK switching this to
reboot=true

**Comment by mpreisler** [comment]
I am ok with merging as is. I am not sure what to do about cases where user can reload it but they need to call something from the command-line.

ACK

**Pull request merged**